react/week2/my-app/src/index.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import Counter from "./counter";
+
+//********************************************************************************************************************************************************* */
+const TodoListItems=({todoitem, ToggleCompleted, DeleteTodo})=>{
+  return <li className={ (todoitem.completed) ? "done": "not-done" }>
+    <div>{todoitem.name}</div>
+    <button onClick={DeleteTodo}>Delete</button>
+    <input type="checkbox" checked={todoitem.completed} onChange={ToggleCompleted}/>
+    <label></label>
```

Comment:
Remember to remove unsued code!

Reply:
About This label, This i am using for the checked and unchecked visual state only.